boardgame-ui/src/components/event/EventAddEditGame.vue

```diff
@@ -182,6 +189,13 @@ export default {
     }
   },
   methods: {
+    setTimeFromDuration(duration) { // arg duration to be provided in minutes
+      duration = Math.ceil(duration / 15) * 15; // get multiple of 15 minutes
```

Comment:
Shouldn't we use a `Math.round()` here ? 
For instance 31min of timer play would result in 45min with `Math.ceil()`.

Reply:
I used `ceil()` on purpose because I think the function `timer.getTotalElapsed()` tends to underestimate the duration of the game (in most games there are some "common" game time, e.g. counting the victory points). But it can be modified if you don't agree :-)

Reply:
Ok, makes sense now. However I feel it is a little counter-intuitive when you encounter this as a user. I think it is better to have `round` and then leave the choice of increasing the duration to the user if necessary. We can brainstorm how to measure overall game duration more accurately for next version. Do you agree ?

Reply:
I think in past events, we tended to round game durations up anyway, so not sure it would annoy users, but since we can indeed change duration manually, it's not that important, and I will change to `round()` :-)

Reply:
Actually I don't think it is annoying and the idea is good. I meant more that it might seem like a bug from the user perspective if it is not documented/explained (especially for short games when the time is like 30 min 10 or something) :)